### PR TITLE
Explore lambda triggers

### PR DESCRIPTION
**Pull request by JCSG99**
ingestion completion now triggers an alarm which invokes the processing lambda

### CODE REVIEW

File: terraform/alarm.tf [hunk 1]
```diff
@@ -34,4 +33,45 @@ resource "aws_sns_topic_subscription" "dummy-reader-topic-subscription" {
...
+  alarm_name          = "ingestion_lambda_complete"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "1"
+  period              = "30"
+  metric_name         = "EventCount"
+  namespace           = "ingestion_lambda_success"
+  threshold           = "1"
+  statistic           = "SampleCount"
+  alarm_actions       = ["arn:aws:sns:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:trigger_processing_lambda_topic"]
+  # alarm actions may need to send an email, or some other way of alerting us to the alarm
```
Comment (Marizzleco) [inline terraform/alarm.tf:58]: Does this comment need to be here?

File: terraform/lambda.tf [hunk 2]
```diff
@@ -41,3 +41,12 @@ resource "aws_cloudwatch_event_target" "ingestion_lambda_target" {
   arn  = aws_lambda_function.ingestion_lambda.arn
   rule = aws_cloudwatch_event_rule.ingestion_lambda_event_rule.name
 }
+
+resource "aws_lambda_function" "processing_lambda" {
+  filename      = "${path.module}/../zips/transformation.zip"
+  function_name = var.processing_lambda_name
+  role          = aws_iam_role.processed-lambda-role.arn
+  handler       = "transformation.tbc"
+  # handler name to be confirmed ^^
```
Comment (Marizzleco) [inline terraform/lambda.tf:50]: And this comment too?

**Review by Marizzleco — COMMENTED** [review]
Just a couple of comments as discussed with Joe :)

File: terraform/alarm.tf (hunk 1 above)
Comment (JCSG99) [reply terraform/alarm.tf:58]: deleted

File: terraform/lambda.tf (hunk 2 above)
Comment (JCSG99) [reply terraform/lambda.tf:50]: yes, as will be changed

**Pull request closed without merging**